.gitignore

```diff
@@ -99,3 +99,4 @@ performance.json
 .history
 
 packages/graphql-toolbox/tests/artifacts
+package-lock.json
```

Comment:
From setting up javascript servers with npm. I noticed that we are not ignoring the package-lock.json files generated by npm, and we probably should to avoid commiting those, if anyone at any point uses npm

Reply:
OK sounds good :+1: